routes/web.php

```diff
@@ -23,6 +23,10 @@
 Route::get('/photo/all', 'PhotoController@all');
 Route::post('/photo/save', 'PhotoController@savePhoto');
 
+Route::get('/token', 'FCMTokenController@index');
+Route::post('/token', 'FCMTokenController@addToken')->middleware('permissions:admin');
```

Comment:
specify middleware in controller contstuctor

```
public function __construct()
    {
        $this->middleware('permissions:admin');

    }
```

Reply:
You told me earlier you want to specify all middleware logic only in routes configuration file